app/users/models.py

```diff
@@ -1,15 +1,17 @@
+import uuid
+
 from sqlalchemy import Column, Integer, String
 from sqlalchemy.dialects.postgresql import UUID
 
-from .database import Base
+from app.database import Base
 
 
 class User(Base):
     __tablename__ = "users"
 
-    id = Column(UUID(as_uuid=True), primary_key=True, index=True)
+    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4(), index=True)
     name = Column(String(100))
     last_name = Column(String(100), nullable=False)
-    user_name = Column(String(100), nullable=False)
+    user_name = Column(String(100), nullable=False, unique=True)
     password = Column(Integer, nullable=False)
```

Comment:
not sure about integer field since passwords generally should be salted or at least hashed.